azure-pipelines.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       pytest-xdist
       scikit-learn
       scipy
-      h5py
+      h5py>=2.10
```

Comment:
Let's leave it as this for now, long term we might want to come back and put minimum pins on stuff here too (although to be honest we'll just pick up the maximum version here anyways, I guess I'm just being too pedantic...)

Reply:
I understand your comment as "h5py>=2.10 is fine for now" so I am not changing it.